app/views/generic_files/upload/_tos_checkbox.html.erb

```diff
@@ -0,0 +1,3 @@
+<label style="display:none;">
+  <%= check_box_tag 'terms_of_service', 1, 1, {data: {activate: 'activate-submit'}} %><strong> I have read and do agree to <%= link_to t('sufia.deposit_agreement'), '/agreement/', target: "_blank" %>.</strong>
```

Comment:
Or it would be if it wasn't for the JS magic that activates the Submit button.  Rather than hacking around that, we'll just leave the hidden checkbox solution in place.